easybuild/easyconfigs/s/SIRIUS/SIRIUS-5.5-CrayIntel-17.08.eb

```diff
@@ -0,0 +1,46 @@
+# created by Anton Kozhevnikov (CSCS)
+easyblock = 'MakeCp'
+
+name = 'SIRIUS'
+version = '5.5'
+
```

Comment:
@toxa81 You might want to add the CUDA version in the configuration file, as we do for other CUDA build. E.g.:
```
cudaversion = 8.0
versionsuffix = '-cuda-%s` % cudaversion
```
You can use the variable `cudaversion` within the configuration file, to avoid hard coded versions.

Reply:
fixed